packages/TelegramClient-Tests.package/TCTUITest.class/instance/forceSaveScreenshotOf.as.in..st

```diff
@@ -0,0 +1,10 @@
+image handling
+forceSaveScreenshotOf: aMorph as: aString in: anotherString
```

Comment:
We could rename aString to fileName and anotherString to folderName

Reply:
Done